JitsiConnection.js

```diff
@@ -22,6 +23,7 @@ export default function JitsiConnection(appID, token, options) {
     this.token = token;
     this.options = options;
     this.xmpp = new XMPP(options, token);
+    this.xmpp.connection.moderator = new Moderator(this.xmpp, this.options);
```

Comment:
Why set it here instead of the `xmpp` or `xmpp.connection` constructor? They have all the necessary fields.